src/test/java/seedu/address/logic/commands/SortPropertyCommandTest.java

```diff
@@ -0,0 +1,134 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.ASC_DEADLINE;
+import static seedu.address.logic.commands.CommandTestUtil.DESC_PROPERTY_NAME;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_SORTING_KEY_PROPERTY_DEADLINE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_SORTING_KEY_PROPERTY_NAME;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_SORTING_ORDER_ASC;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_SORTING_ORDER_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalProperties.getTypicalPropertyBook;
+
+import java.util.Comparator;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.SortPropertyCommand.SortPropertyDescriptor;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.property.Property;
+import seedu.address.testutil.SortPropertyDescriptorBuilder;
+
+
+public class SortPropertyCommandTest {
+
+    private Model model = new ModelManager(getTypicalPropertyBook(), new UserPrefs());
+
+    @Test
+    public void execute_filterByDeadlineInAscendingOrder_success() {
```

Comment:
should it be filterby or sortby? same for the rest below

Reply:
updated